### ISSUE TITLE
Not handling resize

### ISSUE THREAD

**Issue by leobispo** [MurhafSousli/ngx-gallery#67]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?
All
### Repro steps
Try to resize the browser window. It is not rearranging the slider

**Comment by MurhafSousli** [comment]
@leobispo what is the use case?

**Comment by MurhafSousli** [comment]
This is fixed in **v2.0.1**

**Comment by leobispo** [comment]
Perfect, Tnx @MurhafSousli
